mkdocs/docs/2.12.0/litmus-portal-crds-2.12.0.yml

```diff
@@ -801,13 +801,1773 @@ spec:
+                                                        to select'
+                                                      type: string
+                                                  required:
+                                                  - resource
+                                                  type: object
+                                                secretKeyRef:
+                                                  description: Selects a key of a secret
+                                                    in the pod's namespace
+                                                  properties:
+                                                    key:
+                                                      description: The key of the secret
+                                                        to select from.  Must be a valid
+                                                        secret key.
+                                                      type: string
+                                                    name:
+                                                      description: 'Name of the referent.
+                                                        More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                                                        TODO: Add other useful fields.
+                                                        apiVersion, kind, uid?'
+                                                      type: string
+                                                    optional:
+                                                      description: Specify whether the
+                                                        Secret or its key must be defined
+                                                      type: boolean
+                                                  required:
+                                                  - key
+                                                  type: object
+                                              type: object
+                                          required:
+                                          - name
+                                          type: object
+                                        type: array
+                                      hostNetwork:
+                                        description: HostNetwork define the hostNetwork
+                                          of the external pod it supports boolean values
+                                          and default value is false
+                                        type: boolean
                                       image:
+                                        description: Image for the source pod
                                         type: string
-                                        minLength: 1
-                                      hostNetwork:
+                                      imagePullPolicy:
```

Comment:
Add imagePullSecrets as well, refer - https://github.com/litmuschaos/chaos-operator/pull/415/files

Reply:
@ispeakc0de This was added in 2.12.0 release of core release. In 2.12.0 of ChaosCenter, we used 2.11.0. That's why crds of 2.11.0 core components is used here.

Reply:
I have added imagePullSecret in CI crds